operator/builtin/input/file/config.go

```diff
@@ -153,7 +141,7 @@ func (c InputConfig) Build(context operator.BuildContext) ([]operator.Operator,
 		InputOperator:      inputOperator,
 		Include:            c.Include,
 		Exclude:            c.Exclude,
-		SplitFunc:          splitFunc,
+		SplitFunc:          multiline.SplitFunc,
```

Comment:
Would it be better to return `SplitFunc` by `multiline.Build`?

Reply:
Seems ok either way, as long as the error can be surfaced or handled appropriately.